resources/validations/ui/src/context/browser/views/components/SSPReport.tsx

```diff
@@ -34,7 +41,16 @@ const Assertion = ({ assert }: { assert: ValidationAssert }) => {
           )}
           <ul>
             {assert.see && <li>`see: ${assert.see}`</li>}
-            <li>xpath: {assert.location}</li>
+            <li>
+              <a
+                href={href}
+                className="usa-tooltip"
+                data-position="top"
+                title={assert.location}
+              >
+                Show XML context
+              </a>
+            </li>
             <li>test: {assert.test}</li>
```

Comment:
NB: I would also like @rrkane10x to weigh in on whether, at this juncture, we should also include the Schematron test itself. I am not even sure we have a "something else" to put there or we just opt to get rid of it altogether.

Reply:
We could give it the same XML treatment as the SSP... or display some stats on the assertion. I'm in the process of parsing all the SCH into a JS object, so the latter will be more trivial after #109.

Reply:
Ok, makes sense.